server/service/src/rnr_form/insert.rs

```diff
@@ -88,6 +92,7 @@ fn validate(
         CheckOtherPartyType::Supplier,
     )?;
 
+    // TODO... for store! How?
```

Comment:
Programs are masterlists, so they are global.
We could check if it's available it the current store but I don't think it's necessary.

Reply:
While testing the different stores, I realised I could see/search the programs, but that the underlying master_list has store-based visibility. I think I'll add a join on the query....

Reply:
Can I check this is all good with you @jmbrunskill ? [0969e6f](https://github.com/msupply-foundation/open-msupply/pull/4439/commits/0969e6f047ce73a3ab1587ce7302690ea158235f) 🙏

Reply:
Looks good to me! Thanks!